Tests/Editor/CompilationTest.cs

```diff
@@ -30,7 +30,7 @@ public void NoCCUDefines()
 		public void NoEditorVR()
 		{
 			var defines = EditorUserBuildSettings.activeScriptCompilationDefines.ToList();
-			defines.Remove("UNITY_EDITORVR");
+			defines.Remove("UNITY_2017_2_OR_NEWER");

```

Comment:
I wasn't sure at first whether this was doing the job, but it is. The defines list includes the entire list of defines.

That said, I have found some issues in other Unity versions, and am working on a commit to sort them out.